### PR TITLE
Rebase of all commits

### PR DESCRIPTION
**Pull request by wolfiex**
This branch should have a tidied up history.
If you are happy with it I shall use it to replace main.
Will make a backup of the original main into a new branch beforehand.

### CODE REVIEW

**Comment by durack1** [comment]
Might wanna tag this as draft until it's a little cleaner, e.g. `CVs/CMIP6Plus_CV_CMIP6Plus_CV_cleanup.json.json`

Ah so close. Give me a second. I will overwrite it.

File: CVs/CMIP6Plus_CV.json [hunk 1]
```diff
@@ -1063,16 +1063,16 @@
             }
         },
         "source_type": [
```
Comment (durack1) [inline CVs/CMIP6Plus_CV.json:1065]: May as well alphabetize this, no?

File: CVs/CMIP6Plus_CV_cleanup.json [hunk 2]
```diff
@@ -0,0 +1,1184 @@
+{
```
Comment (durack1) [inline CVs/CMIP6Plus_CV_cleanup.json:1]: It is not obvious to me why this has `*_cleanup.json` appended. It seems that https://github.com/WCRP-CMIP/CMIP6Plus_CVs/tree/main/CVs/CMIP6Plus_CV.json would the correct name, or am I missing something?

File: CVs/CMIP6Plus_CV_cleanup.json [hunk 3]
```diff
@@ -0,0 +1,1184 @@
...
+            "esm-hist": {
+                "activity_id": [
+                    "CMIP"
+                ],
+                "additional_allowed_model_components": [
+                    "AER",
+                    "CHEM"
+                ],
+                "description": "CMIP6 historical (CO2 emission-driven)",
+                "end": "2014",
```
Comment (durack1) [inline CVs/CMIP6Plus_CV_cleanup.json:115]: Just thinking out aloud here, 2014 was the CMIP6 era end date, for CMIP6Plus we're hoping for 2021 to start (new forcings) and then this to continue to extend, so how would we capture this info, when the start date is fixed but the end might be 2021, 2022, 2023, 2024, ...

File: CVs/CMIP6Plus_CV_cleanup.json [hunk 4]
```diff
@@ -0,0 +1,1184 @@
...
+                "activity_id": [
+                    "LESFMIP"
+                ],
+                "additional_allowed_model_components": [
+                    "AER",
+                    "CHEM",
+                    "BGC"
+                ],
+                "description": "As hist-aer but with updated forcings",
+                "end": 2024,
```
Comment (durack1) [inline CVs/CMIP6Plus_CV_cleanup.json:203]: This is also likely to extend as per Doug's discussions - they envisage these experiments to be "operational" and will continue to be extended as new forcings become available

**Review by durack1 — COMMENTED** [review]
@wolfiex I think continue to merge toward finalizing this - we may need some tweaks as we go, and these targets continue to shift

File: CVs/CMIP6Plus_CV.json (hunk 1 above)
Comment (wolfiex) [reply CVs/CMIP6Plus_CV.json:1065]: Good point that was on a to-do. "Sorted" now.

File: CVs/CMIP6Plus_CV_cleanup.json (hunk 2 above)
Comment (wolfiex) [reply CVs/CMIP6Plus_CV_cleanup.json:1]: cleanup is the branch. The main branch just has no branch extension and removes all the additional ones. 
This is so users can compare their branch to the main one without causing multiple conflicts (e.g. when adding lots of sources (via branches) at once.

File: CVs/CMIP6Plus_CV_cleanup.json (hunk 3 above)
Comment (matthew-mizielinski) [reply CVs/CMIP6Plus_CV_cleanup.json:115]: Could leave it blank? Would be tempted to wait until you have a confirmed set of forcings for new experiments. Then could add extra guidance -- remember that this doesn't get used by CMOR.

Comment (durack1) [reply CVs/CMIP6Plus_CV_cleanup.json:115]: > Could leave it blank? Would be tempted to wait until you have a confirmed set of forcings for new experiments. Then could add extra guidance -- remember that this doesn't get used by CMOR.

Forcings are starting to drop now, got 3 of ~8+ - ending in 2021 to start, and then extending to ~2024

**Review by matthew-mizielinski — APPROVED** [review]
LGTM, behaves as expected with regard to producing files.  Null entries in CVs give some error messages from CMOR, but do not cause problems -- I presume these fields will be sorted as part of the release process

**Pull request merged**